src/main/scala/io/iohk/ethereum/jsonrpc/RawTransactionCodec.scala

```diff
@@ -0,0 +1,15 @@
+package io.iohk.ethereum.jsonrpc
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.SignedTransaction
+import io.iohk.ethereum.network.p2p.messages.CommonMessages.SignedTransactions.SignedTransactionEnc
+import io.iohk.ethereum.rlp
+
+object RawTransactionCodec {
+
+  def rawTransactionFromBlock(blockTxs: Seq[SignedTransaction], index: Int): Option[ByteString] =
+    blockTxs.lift(index).map(asRawTransaction)
+
+  def asRawTransaction(e: SignedTransaction): ByteString =
```

Comment:
I would remove method above and use this one on rpc codec level instead of service method. In this way all `Raw` methods can be implemented in a fashion like this:
```
def getRawTxByHash(request) = getTransactionByHash().map(_.map(RawTransactionResponse(_))
```

Reply:
I have done sth like that here: https://github.com/input-output-hk/mantis/pull/720/commits/0f576e8df0608e4259164689293e3e50ae7c6986

I like the fact that byte operations are not handled by this implicit in Json and service return domain object SealedTransaction. Nice idea 🙇